test/unit/serializer/basic_types.d

```diff
@@ -0,0 +1,123 @@
+module serializer.basic_types;
+
+import dmd.root.serializer : serialize;
+
+@("bool, true")
+unittest
+{
+    const result = serialize(true);
+    assert(result == "--- true", result);
+}
```

Comment:
All these unittest test just _one_ function which doesn't need any state. What's the motivation of not using embedded `unittest`s within DMD as they would automatically document the function whereas these tests might be a lot harder to find to a reader.

Reply:
* I'm pretty sure you said somewhere that those tests are not being run/are not passing on Windows
* I think this unit test runner is better, it allows to run only a single unit test block
* I think there will be too many unit tests in one place